Adapter/PlentymarketsAdapter/ResponseParser/Product/ProductResponseParser.php

```diff
@@ -318,6 +320,10 @@ private function getPrices(array $variation)
                 $pseudoPrice = (float) $priceArray['rrp']['price'];
             }
 
+            if (isset($priceArray['specialOffer'])) {
+                $pseudoPrice = (float) $priceArray['specialOffer']['price'];
```

Comment:
I think this is not correct. Special prices ("Aktionspreise") are lower than the default price (or rrp/UVP). Yet, the pseudo price must be higher than the default price, otherwise it is ignored. Therefore, the special price must be assigned to the price and the price must become the pseudo price, if the pseudo price is not set by the rrp.